docs/cody/clients/install-cli.mdx

```diff
@@ -0,0 +1,212 @@
+
+```powershell
+Install-Module -Name CredentialManager
+```
+
+  </Tab>
+  <Tab title="Linux (Ubuntu)">
+The [`secret-tool`](https://manpages.ubuntu.com/manpages/focal/man1/secret-tool.1.html#:~:text=secret%2Dtool%20is%20a%20command,of%20attribute%20keys%20and%20values) command-line tool is used. Run the following commands to install it:
+```shell
+sudo apt install libsecret-tools
+sudo apt install gnome-keyring
+```
+  </Tab>
+</Tabs>
+
+Run the following command to authenticate the Cody CLI:
+
+<Tabs>
+  <Tab title="Browser (fastest)">
+```shell
+cody auth login --web
+```
+This will open a browser window where you can authenticate with your Sourcegraph account.
+Close the browser tab after authentication is complete.
+  </Tab>
+  <Tab title="Command Line">
+- For Cody Pro/Free accounts, create an access token at https://sourcegraph.com/user/settings/tokens.
+- For Cody Enterprise accounts, sign into your Sourcegraph Enterprise account and create an access token under `Account > Settings > Access Tokens`.
+```shell
+export SRC_ENDPOINT=ENDPOINT
+export SRC_ACCESS_TOKEN=ACCESS_TOKEN
+cody auth login # stores the access token securely
+```
+You don't need to export `SRC_ENDPOINT` or `SRC_ACCESS_TOKEN` after running `cody auth login`.
+Subsequent command invocation will use the stored access token.
+  </Tab>
+</Tabs>
+
+Confirm that the authentication was successful by running the following command:
+
+```shell
+cody auth whoami
+# ✔ Authenticated as USERNAME on ENDPOINT
+```
+
+## Authenticate with only environment variables
+
+Skip this step if you have already authenticated with the `cody auth login` command.
+
+If you prefer not to let Cody CLI store your access token, you can also pass the endpoint URL and access token through the environment variables `SRC_ENDPOINT` and `SRC_ACCESS_TOKEN`.
+
+<Tabs>
+  <Tab title="macOS/Linux">
+```shell
+export SRC_ENDPOINT=ENDPOINT
+export SRC_ACCESS_TOKEN=ACCESS_TOKEN
+```
+  </Tab>
+  <Tab title="Windows">
+```powershell
+$env:SRC_ENDPOINT = "ENDPOINT"
+$env:SRC_ACCESS_TOKEN = "ACCESS_TOKEN"
+```
+  </Tab>
+</Tabs>
+
+It's recommended to store these access tokens in a secure location. 
+For example, you can store them with a password manager like [1Password](https://1password.com/) or [Bitwarden](https://bitwarden.com/).
+
+It may be tempting to export these variables in your shell startup script, but this is not recommended because it will expose your access token to all commands you run from the terminal.
+Instead, consider sourcing these environment variables on-demand when you need to authenticate with the Cody CLI.
+
+## Sign out with cody auth logout
+
+To sign out of the Cody CLI, run the following command:
+
+```shell
+cody auth logout
+```
+
+Running this command wil remove the access token from the secure storage of your operating system.
+
+Run `cody auth whoami` to confirm that the Cody CLI is no longer authenticated.
+
+# Chat with Cody CLI
```

Comment:
```suggestion
## Chat with Cody CLI
```
Only one H1 per page. Rest should be H2s or H3s. :)

Reply:
Fair enough. I've written a lot of docs with Docusaurus in the past where H2/H3 appear in the sidebar, not the H1. It's useful to be able to group sidebar items into sections like installation, chat, ... The sidebar looks like this now

![CleanShot 2024-07-18 at 14 27 41@2x](https://github.com/user-attachments/assets/d3ae571d-1be3-4370-bb12-d38917dc8b80)